docs/ReleaseNotes.md

```diff
@@ -28,7 +28,7 @@ The Guava dependency version has been updated to 31.1. Projects may need to chec
 * **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** https://github.com/FoundationDB/fdb-record-layer/issues/1985 [(Issue #1985)](https://github.com/FoundationDB/fdb-record-layer/issues/1985)
```

Comment:
Could this include a description instead of the URL? Something like "Avoid planning IN-predicates as IN-joins when the IN predicate is not optimized by an index" may be good to have to make the release notes a bit more readable

Reply:
oh a cp went wrong